### PR TITLE
perf(gen): Hashed data - pkinit, rt*, safeq*

### PR DESCRIPTION
**Pull request by cuadradek**
Services pkinit, rt, rt_bbmri, safeq and safeq5 are now using
getHashedHierarchicalData.

### CODE REVIEW

**Review by Johaney-s — APPROVED** [review]
looks good

**Review by metodej — APPROVED** [review]
It looks great, I have just a nitpick I think it would be nice to be consistent in parameters ordering in `get*AttributeValue` methods...

File: gen/safeq [hunk 1]
```diff
@@ -141,25 +136,32 @@ perunServicesInit::finalize;
...
-	my $memberAttributes = shift;
-	my $resourceAttributes = shift;
-	my $login = $memberAttributes->{$A_USER_LOGIN};
-
-	unless(defined $usersByLogin->{$memberAttributes->{$A_USER_LOGIN}}) {
-	my %tmp =  ( %$memberAttributes, %$resourceAttributes );
-		$usersByLogin->{$memberAttributes->{$A_USER_LOGIN}} = $memberAttributes;
-		$usersByLogin->{$memberAttributes->{$A_USER_LOGIN}}->{$A_R_PREPAID} = $resourceAttributes->{$A_R_PREPAID};
-		$usersByLogin->{$memberAttributes->{$A_USER_LOGIN}}->{$A_R_PRINT_FOR_FREE} = ( $resourceAttributes->{$A_R_PRINT_FOR_FREE} || 0) ;
+	my $memberId = shift;
```
Comment (zlamalp) [inline gen/safeq:139]: See where you print `role:` in LDIF file. You can probably use `sort keys ...`  to get comparable outputs. It's same for `billingcode:` or `card:` fields.

**Review by zlamalp — CHANGES_REQUESTED** [review]

File: gen/safeq (hunk 1 above)
Comment (zlamalp) [reply gen/safeq:139]: I'm waiting for confirmation from service owner that we can sort those attributes, so please wait with that cahnge. Also it seems, that `safeq5` version can be deleted instead - also waiting for confirmation, so we won't merge this yet.

**Pull request merged**